02.calendar/calendar.js

```diff
@@ -0,0 +1,31 @@
+import { DateTime } from "luxon";
+import minimist from "minimist";
+
+const argv = minimist(process.argv.slice(2));
+const today = DateTime.now();
+
+const year = parseInt(argv.y) || today.year;
+const month = parseInt(argv.m) || today.month;
+
+const firstDate = DateTime.fromObject({
+  year: year,
+  month: month,
+});
+const firstDay = firstDate.weekday;
+
+console.log(`      ${firstDate.month}月 ${firstDate.year}`);
+console.log("日 月 火 水 木 金 土");
+if (firstDate.weekday !== 7) {
+  process.stdout.write("   ".repeat(firstDay));
+}
+for (let day = 1; day <= firstDate.daysInMonth; day++) {
+  const datetime = firstDate.set({ day });
```

Comment:
```suggestion
  const currentDate = firstDate.set({ day });
```

timeはカレンダーで利用しないので、timeを含まない名前の方がいいかも、と思いました

Reply:
[🐛 fix: より適した変数名に変更](https://github.com/reckyy/js-practices/pull/2/commits/3e9c703bcb302234ead7398c12d48fe999ea3b63)